rust/crates/nasl-function-proc-macro/src/codegen.rs

```diff
@@ -195,6 +195,7 @@ impl<'a> ArgsStruct<'a> {
         let mangled_ident = Ident::new(&mangled_name, ident.span());
         let inner_call = self.get_inner_call_expr(&mangled_ident, asyncness);
         quote! {
+            #[allow(clippy::too_many_arguments)]
```

Comment:
I added this because we cannot really do anything about the number of arguments in our builtin functions anyways and without it adding it in the macro, there is no way for the user of `nasl_function!` to add this `#[allow(...)]` attribute themselves.